tests/prague/eip663_dupn_swapn_exchange/__init__.py

```diff
@@ -0,0 +1,16 @@
+"""
+abstract: Tests [EIP-663: SWAPN, DUPN and EXCHANGE instructions](https://eips.ethereum.org/EIPS/eip-663)
+    Tests for [EIP-663: SWAPN, DUPN and EXCHANGE instructions](https://eips.ethereum.org/EIPS/eip-663).
+"""  # noqa: E501
+
+import pytest
+
+from ethereum_test_tools import Transaction
+
+REFERENCE_SPEC_GIT_PATH = "EIPS/eip-663.md"
+REFERENCE_SPEC_VERSION = "6aed382f86258f33603f5dc275956f739aaae096"
+
+
+@pytest.fixture
```

Comment:
For fixtures that are used in all tests, these can be added to a file called `conftest.py`.

Example: https://github.com/ethereum/execution-spec-tests/blob/main/tests/cancun/eip4844_blobs/conftest.py

Then there is no need to manually importing `tx` in each test file.